bin/release-plugin-to-dot-org.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+CURRENT_PATH=`pwd`
+
+# Pull down the SVN repository.
+echo "Pulling down the SVN repository for woocommerce-admin"
+SVN_WOOCOMMERCE_ADMIN_PATH=/tmp/woocommerce/svn-woocommerce-admin
+svn co https://plugins.svn.wordpress.org/woocommerce-admin/ $SVN_WOOCOMMERCE_ADMIN_PATH
+cd  $SVN_WOOCOMMERCE_ADMIN_PATH
+
+# Get the tagged version to release.
+echo "Please enter the version number to release to woocommerce.org, for example, 1.0.0: "
```

Comment:
@becdetat I just noticed, i think this should say "wordpress.org"?

Reply:
totally